cli/cli.go

```diff
@@ -134,6 +126,29 @@ func (c *CLI) Upgrade() error {
 	return update.Apply(bin, update.Options{})
 }
 
+func (c *CLI) fetchLatestRelease() error {
+	resp, err := HTTPClient.Get(LatestReleaseURL)
+	if err != nil {
+		return err
+	}
+	defer resp.Body.Close()
+
+	if resp.StatusCode > 399 {
```

Comment:
Ah, now I remember why. I wanted to add in an extra error that could show all the response headers in the output if we make a successful call, but the API is giving us a less-than-OK answer. E.g. we hit the rate limit. Adding all that extra error code seemed super gross up in the original method.